pandas/_libs/tslibs/util.pxd

```diff
@@ -1,10 +1,18 @@
-from numpy cimport ndarray
-cimport numpy as cnp
-cnp.import_array()
 
-cimport cpython
 from cpython cimport PyTypeObject
 
+cdef extern from *:
```

Comment:
we don't need this in util.pxd? (the higher level one)?

Reply:
does this change perf?

Reply:
> does this change perf?

No.

> we don't need this in util.pxd? (the higher level one)?

a) everything in this file is cimported into the other one, so it is available there.   b) No, this is only used by tslibs.period.